tools/py_common/btree_format.py

```diff
@@ -178,26 +194,92 @@ def parse(b: binary_data.BinaryFile, disagg = False) -> 'BlockHeader':
         '''
         # WT_BLOCK_HEADER in block.h (12 bytes)
         h = BlockHeader()
-        if disagg:
-            # Disagg sets additional fields.  If they are examined
-            # by non-disagg code, an exception will be thrown (by design).
-            h.disagg_magic = b.read_uint8()
-            h.disagg_version = b.read_uint8()
-            h.disagg_compatible_version = b.read_uint8()
-            h.disagg_header_size = b.read_uint8()
-            h.checksum = b.read_uint32()
-            h.disagg_previous_checksum = b.read_uint32()
-            h.disagg_reconciliation_id = b.read_uint8()
-            h.flags = b.read_uint8()
-            h.unused = int.from_bytes(b.read(2), byteorder='little')
-        else:
-            h.disk_size = b.read_uint32()
-            h.checksum = b.read_uint32()
-            h.flags = b.read_uint8()
-            h.unused = int.from_bytes(b.read(3), byteorder='little')
+        h.disk_size = b.read_uint32()
+        h.checksum = b.read_uint32()
+        h.flags = BlockFlags(b.read_uint8())
+        h.unused = int.from_bytes(b.read(3), byteorder='little')
         return h
+    
+    def __str__(self):
+        header_string = (
+            f"Block Disagg Header:\n"
+            f"  disk_size: {str(self.disk_size)}"
+            f"  checksum: {str(self.checksum)}"
+            f"  flags: {str(self.flags)}"
+        )
+        return header_string
+
+class BlockDisaggFlags(enum.IntFlag):
+    '''
+    Disagg block flags from block.h
+    '''
+    WT_BLOCK_DISAGG_DATA_CKSUM = 0x1
```

Comment:
I think it's ok to keep the full word
```suggestion
    WT_BLOCK_DISAGG_DATA_CHECKSUM = 0x1
```

Reply:
I would like to keep this the same as in WT so it can be easily searched